imap_processing/tests/idex/test_idex_l1b.py

```diff
@@ -0,0 +1,117 @@
+"""Tests the L1b processing for IDEX data"""
+
+import numpy as np
+import pandas as pd
+import pytest
+import xarray as xr
+
+from imap_processing import imap_module_directory
+from imap_processing.cdf.utils import write_cdf
+from imap_processing.idex.idex_l1b import idex_l1b, unpack_instrument_settings
+
+
+@pytest.fixture()
```

Comment:
I suggest changing the scope of this fixture to "module". That way the fixture is only executed once and the return value gets cached so that `idex_l1b` doesn't have to run for each test that calls this fixture.
```suggestion
@pytest.fixture(scope="module")
```
In case you want to know more about fixture scopes: https://docs.pytest.org/en/stable/how-to/fixtures.html#scope-sharing-fixtures-across-classes-modules-packages-or-session